src/constants/index.ts

```diff
@@ -0,0 +1,5 @@
+const NO_WEB3 = "Web3 not enabled or available.";
+const NO_ACCOUNT = "No accounts available";
+const CRYPTO_LOAD_FAIL = "Crypto wait ready failed";
```

Comment:
I made this update.
```
const NO_WEB3 = new Error("Web3 not enabled or available.");
const NO_ACCOUNT = new Error("No accounts available");
const CRYPTO_LOAD_FAIL = new Error("Crypto wait ready failed");
```
I created these for some specific scenarios that do not throw errors, but I wonder if this is the correct usage, for example, if web3 is enabled but the user does not have accounts available:

  ```
 const allWeb3Accounts = await web3Accounts(accountSettings);
 if (!allWeb3Accounts || allWeb3Accounts.length === 0) throw NO_ACCOUNT;

```

Reply:
No, you want to create an instance of an error where you throw it since the backtrace will point to the line where instance was created, not where it was thrown from.

Reply:
[#de61605ed2dd246a99ef167984ec4317292d661f](https://github.com/subspace/subspace.js/pull/3/commits/de61605ed2dd246a99ef167984ec4317292d661f)